### PR TITLE
Update cryptography 36.x resources

### PR DESCRIPTION
**Pull request by alex**
This ensures that all brew packages use the latest version in the 36.x series.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by alex** [comment]
Looks like this hit a timeout.

**Comment by iMichka** [comment]
Hi. Thanks for your contribution. Yes, we have a one hour timeout. Let me re-run this.

File: Formula/dvc.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Dvc < Formula
   url "https://files.pythonhosted.org/packages/d2/f7/88b9578181fc14a812a95e9fc92cfa2da8e4e857a4b247af36bf85e0a3d8/dvc-2.9.3.tar.gz"
   sha256 "513d0d9ee3c3123e616f760eb8556ee569121286c3e400a1fe4b3bf66e6dea29"
   license "Apache-2.0"
+  revision 1
```
Comment (carlocab) [inline Formula/dvc.rb:9]: This is a rev-bump only change. Is that intentional?

**Review by carlocab — COMMENTED** [review]
Can you split your single commit into 5, one for each formula file modified? Use the commit message
```
<formula name>: update cryptography resource
```
Thanks!

File: Formula/dvc.rb (hunk 1 above)
Comment (alex) [reply Formula/dvc.rb:9]: Hah, I believe that it got updated between when I originally sent the PR and when I rebased.

**Comment by alex** [comment]
Will split up the commits now.

**Review by branchv — APPROVED** [review]
@alex thank you!

**Review by carlocab — APPROVED** [review]
Thanks!

If we're going to keep bumping `cryptography` independently of version bumps I wonder if we should just have it as a separate formula. We'd probably need formulae for `cffi` and `pycparser`, though.

**Comment by alex** [comment]
Unfortunately we'd probably end up needing a handful of formulas, as there's many versions in use ATM:

```
/o/h/L/T/h/homebrew-core ❯❯❯ rg --no-filename --no-line-number -o 'cryptography-.*?.tar.gz' | sort | uniq -c | sort
   1 cryptography-2.1.4.tar.gz
   1 cryptography-3.4.5.tar.gz
   1 cryptography-3.4.tar.gz
   2 cryptography-2.8.tar.gz
   2 cryptography-2.9.2.tar.gz
   3 cryptography-3.3.2.tar.gz
   4 cryptography-36.0.0.tar.gz
   5 cryptography-3.4.8.tar.gz
   5 cryptography-36.0.1.tar.gz
   7 cryptography-3.4.7.tar.gz
   8 cryptography-3.4.6.tar.gz
   8 cryptography-35.0.0.tar.gz
```

My goal here is really just to consolidate these, then work with upstreams to get version caps removed.

**Pull request closed without merging**